BasicsCodelab/app/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation "androidx.compose.ui:ui:$compose_version"
+    implementation "androidx.activity:activity-compose:1.3.0-SNAPSHOT"
```

Comment:
@yrezgui is this something we need people to manually add for the `alpha12` release until we fix the AS template?

Reply:
Yes, I've added to the doc